apps/server/src/modules/user-login-migration/service/school-migration.service.ts

```diff
@@ -78,6 +78,21 @@ export class SchoolMigrationService {
 		await this.userService.saveAll(notMigratedUsers.data);
 	}
 
+	async restartMigration(schoolId: string): Promise<void> {
+		const school: SchoolDO = await this.schoolService.getSchoolById(schoolId);
+		const migratedUsers: Page<UserDO> = await this.userService.findUsers({
```

Comment:
This looks like a generic findUser not a special method that need no knowlege to use. A outdatedSinceEquals is a special case. It should be a unqiue operation. 

It make also no sense that the service return a paginated result with data key. This should be changed. See comment for pagination.

Reply:
Should be solved with removing the unnessary "outdatedSinceEquals"